Content.Server/DeltaV/StationEvents/Events/GlimmerMiteSpawnRule.cs

```diff
@@ -0,0 +1,58 @@
+using System.Linq;
+using Robust.Shared.Random;
+using Content.Server.GameTicking.Rules.Components;
+using Content.Server.NPC.Components;
+using Content.Server.Psionics.Glimmer;
+using Content.Server.StationEvents.Components;
+using Content.Shared.Psionics.Glimmer;
+using Content.Shared.Abilities.Psionics;
+
+namespace Content.Server.StationEvents.Events;
+
+internal sealed class GlimmerMiteRule : StationEventSystem<GlimmerMiteRuleComponent>
+{
+    [Dependency] private readonly IRobustRandom _robustRandom = default!;
+    [Dependency] private readonly GlimmerSystem _glimmerSystem = default!;
+
+    private static readonly string MitePrototype = "MobGlimmerMite";
```

Comment:
this should be stored on the rule component + then rename it so it can be reused by other rules

```cs
[DataField(required: true)]
public EntProtoId MobPrototype;
```

then something like this

```cs
- type: GlimmerMobRule
  mobPrototype: MobGlimmerMite
```

Reply:
So with this one, do I move these four lines of code over to the rule component add those other things you commented?

Reply:
just remove the `private static readonly string MitePrototype = "MobGlimmerMite";` line and after the component has stuff use `rule.MobPrototype` (or whatever the rule component is named)